src/minimega/vm_cli.go

```diff
@@ -583,10 +584,41 @@ Clear all tags from all VMs:
 
         clear vm tag all`,
 		Patterns: []string{
-			"clear vm tag <vm id or name or all> [tag]",
+			"clear vm tag",
+			"clear vm tag <vm target> [tag]",
 		},
 		Call: wrapSimpleCLI(cliClearVmTag),
 	},
+	{ // vm target (dummy command)
```

Comment:
@floren 

This is an interesting approach, but does this add confusion to someone looking at the output of 'help'? Do they initially see 'vm target' as a command?

Do we accomplish the same thing by having the following one-liner in the help text of each command that has the 'target' syntax:

"vm target can be any of a VM name, ID, 'all', or a range such as foo,bar[1-3]"